include/zephyr/rtio/rtio.h

```diff
@@ -286,6 +340,17 @@ struct rtio_sqe {
 
 		/** OP_I2C_CONFIGURE */
 		uint32_t i2c_config;
+
+		/** OP_I3C_CONFIGURE */
+		struct {
+			/* enum i3c_config_type type; */
+			int type;
+			void *config;
+		} i3c_config;
+
+		/** OP_I3C_CCC */
+		/* struct i3c_ccc_payload *ccc_payload; */
+		void *ccc_payload;
```

Comment:
Is the payload mutable? Should potentially be const?

Reply:
yes... there is a num_xfer with in there to return the 'true' number of bytes actually sent/received as the target can control when the End of Data occurs.... but I should probably remove this as I was trying to integrate the `do_ccc` in to the rtio... but gave up for now

Reply:
This sounds like some more fun things to look at in the future for sure.

Reply:
yeah... thinking more about it I think i'll just keep it in for now.